runtime/jnichk/jnicheck.c

```diff
@@ -79,6 +79,7 @@ static IDATA jniCheckParseOptions(J9JavaVM* javaVM, char* options);
 static IDATA jniCheckProcessCommandLine(J9JavaVM* javaVM, J9VMDllLoadInfo* loadInfo);
 static UDATA globrefHashTableHashFn(void *entry, void *userData);
 static UDATA globrefHashTableEqualFn(void *leftEntry, void *rightEntry, void *userData);
+static void jniCheckJClassSubclass(JNIEnv* env, const char* function, IDATA argNum, jobject aJobject, J9Class* expectedSupclass, const char* expectedType);
```

Comment:
To make it clearer, you should use jclass instead of jobject (and change the parameter name).

Reply:
Replaced `jobject aJobject` with `jclass aJclass`, please have another look.